frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -80,7 +81,7 @@ const filterByArchAndOS = (items: OperatorHubItem[]): OperatorHubItem[] => {
 
     return (
       _.includes(relevantLabels.os, targetGOOSLabel) &&
-      _.includes(relevantLabels.arch, targetGOARCHLabel)
+      _.some(relevantLabels.arch, (arch) => _.includes(targetNodeArchitecturesLabels, arch))
```

Comment:
```suggestion
      relevantLabels.arch.some((arch) => _.includes(targetNodeArchitecturesLabels, arch))
```

Reply:
FWIW, I used `_.some` here because the existing code is already using Lodash.

Reply:
I'll leave it up to you if you want to get rid of some lodash here.